taskcluster/ci/toolchain/android.yml

```diff
@@ -23,3 +22,31 @@ linux64-android-sdk-linux-repack:
         toolchain-alias: android-sdk-linux
     treeherder:
         symbol: TL(android-sdk-linux)
+    worker:
+        docker-image: {in-tree: base}
+
+
+linux64-android-gradle-dependencies:
+    description: "Android Gradle dependencies toolchain task"
+    fetches:
+        toolchain:
+            # Aliases aren't allowed for toolchains depending on toolchains.
+            - linux64-android-sdk-linux-repack
+    run:
+        script: android-gradle-dependencies.sh
```

Comment:
Unlike what we have in central, I couldn't use the `misc` folder https://searchfox.org/mozilla-central/source/taskcluster/scripts/misc. Taskgraph couldn't find `android-gradle-dependencies.sh` there, but it somehow does in central https://searchfox.org/mozilla-central/rev/2cd2d511c0d94a34fb7fa3b746f54170ee759e35/taskcluster/ci/toolchain/android.yml#51.

See https://firefox-ci-tc.services.mozilla.com/tasks/DRphY83CQqyirGUIc0X6wA/runs/0/logs/https%3A%2F%2Ffirefox-ci-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FDRphY83CQqyirGUIc0X6wA%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log#L3009